test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -280,7 +286,14 @@
           [foo-service baz-service fail-service]
           (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
             (is (= 503 (:status resp)))
-            (is (= "error" (slurp (:body resp)))))))
+            (is (= "unknown" (slurp (:body resp)))))))
```

Comment:
@aperiodic The docs, a docstring, and the code all say something different about how this is supposed to work; i was just wondering if you remembered which was the "right" thing. i assume the code as it stands should be taken as definitive and the docstring/docs updated.